packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/testCorrectChangeListItemsSelectedOnInitialization.st

```diff
@@ -0,0 +1,4 @@
+testing-functionality
+testCorrectChangeListItemsSelectedOnInitialization
+	
+	self assert: (self changesBrowser historyListSelectionIndex = 0)
```

Comment:
I am *always* confused: Is it `=` or `==`?

Reply:
Usually =